### ISSUE TITLE
react-application-alerts change style.

### ISSUE THREAD

**Issue by Krodgersminto** [pnp/sp-starter-kit#687]
Hi I am new to sharepoint development and was wondering for the react-application-alerts extension is it possible to change the style and change the font size, background color, animation effects. I have looked at the documention and it seems possible just not sure where to start.

Thank you

**Comment by eoverfield** [comment]
Yes, this should be possible. You'll need to clone the repo, then review the codebase here: https://github.com/pnp/sp-starter-kit/tree/master/source/react-application-alerts.

From there you should be able to run your standard build pipeline:
npm install
gulp serve
gulp bundle
gulp package-solution

as you would with any other SPFx solution. If you are new to SPFx, I suggest you start with general SPFx onboarding from https://learn.microsoft.com/en-us/sharepoint/dev/spfx/sharepoint-framework-overview, then you should have no problem updating this extension to match your requirements.
